### PR TITLE
JBPM-5207 - process creation improvements in designer

### PR DESCRIPTION
**Pull request by mswiderski**
- enable svg save by default,
- set packageName in process according to location in project so svg files are created in proper location

@tsurdilo @jlindop could you please review this?

### CODE REVIEW

**Comment by tsurdilo** [comment]
+1

File: jbpm-designer-backend/src/test/java/org/jbpm/designer/server/DefaultDesignerAssetServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+import org.mockito.runners.MockitoJUnitRunner;
+import org.uberfire.backend.vfs.Path;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+
+@RunWith(MockitoJUnitRunner.class)
+public class DefaultDesignerAssetServiceTest {
```
Comment (jomarko) [inline jbpm-designer-backend/src/test/java/org/jbpm/designer/server/DefaultDesignerAssetServiceTest.java]: @mswiderski could you please merge this test class with test classes in package https://github.com/jomarko/jbpm-designer/tree/master/jbpm-designer-backend/src/test/java/org/jbpm/designer/service ?

Comment (mswiderski) [inline jbpm-designer-backend/src/test/java/org/jbpm/designer/server/DefaultDesignerAssetServiceTest.java]: sure thing, thanks for letting me know about that location, will update PR and drop a comment when ready

**Comment by mswiderski** [comment]
@jomarko PR updated, is that what you had in mind?

**Comment by jomarko** [comment]
@mswiderski yes, my point was: not introduce next test class for DefaultDesignerAssetService and you accomplished it. Maybe it would be also worth to move test package "org.jbpm.designer.service" into "org.jbpm.designer.server.service" - that corresponds to package of DefaultDesignerAssetService.

**Comment by mswiderski** [comment]
moved the package, @jomarko please double check it

**Comment by jomarko** [comment]
@mswiderski definitive +1, thanks

**Comment by mswiderski** [comment]
jenkins retest this

**Pull request merged**